### DISCUSSION TITLE
Request cannot be completed; no file at `xxxx/xxx/tableOne-2024-08-08_final.tex`.

### DISCUSSION THREAD

**Discussion by santsang** [jupyterlab/jupyterlab-latex#241] [Q&A]
Hi

I am so excited to find this extension.

However, I go no clue do I get the extension to recognise the file is a latex file.

Thank you very much for your advice.

Sandro

**Comment by ktaletsk** [comment]
Hi @santsang , could you provide more details on what you have been doing? What did you clicked on, where did the error message appeared? What version of JupyterLab, this extension you use? What is your `tex` compiler?

**Comment by santsang** [comment]
Hi

I follow the instruction.  I open the file in jupyter-lab.  Then I right click it...  Then I got this message.  Perhaps I need to install the minimal latex?  I am not sure how it works.  Since we can type latex in markdown cell, I expect that install minimal latex is not neccessary.  Your advice please.  Sandro

**Reply by ktaletsk** [reply]
@santsang please take a look at the requirements section: https://github.com/jupyterlab/jupyterlab-latex?tab=readme-ov-file#requirements
In particular, you must have
> An application that can compile .tex files to PDF (e.g., pdflatex, xelatex; use pdflatex.exe on Windows with MiKTeX). This application must be available as a command in the same environment as the notebook server.

There are many options for different TeX compilers, so we don't install it for you.
